strings/base_string.h

```diff
@@ -326,6 +326,38 @@ WINRT_EXPORT namespace winrt
         {
             return rend();
         }
+        
+        inline bool starts_with(const wchar_t value) const noexcept
+        {
+            return empty() ? false : data()[0] == value;
+        }
+
+        inline bool ends_with(const wchar_t value) const noexcept
+        {
+            return empty() ? false : data()[size() - 1] == value;
+        }
```

Comment:
Given that these are also new to C++20, it might be simpler to just lump them together and use string_view to implement them all.